### ISSUE TITLE
RFC: Deprecate reactxp-navigation -- or find a maintainer!

### ISSUE THREAD

**Issue by fbartho** [microsoft/reactxp#1234]
The navigation extension hasn't seen much maintenance for a *long* time. #1027 

Additionally, these days, there are other libraries that provide much more full-featured Navigation Libraries -- For example, reactnavigation.org [supports both react-native and react-native-web](https://reactnavigation.org/docs/web-support/).

Is this something that we feel has high value for being part of ReactXP? Anybody at Microsoft?

One concern I have on my mind is Windows & macOS support, as I don't develop on Windows. I think react-navigation should work on macOS & Windows by virtue of those platforms using import-aliasing to swap out the `react-native` import. The tight coupling there would be undesirable in my projects/company (which pushed us towards reactxp) but it might have a side-benefit of widening the community of supported libraries.

### Options

A. Continue status quo, with minimal maintenance. :'(
B. Find a maintainer! (Maybe somebody who uses it in-house?)
C. Deprecate reactxp-navigation and point people in the direction of other navigation library options.
D. Find people to evangelize ReactXP to other libraries, and convince them to integrate it directly (seems hard)

If you read this ticket please speak up as to your preferences! Absent other input, I'm leaning towards "Option C: Deprecate"

**Comment by mikehardy** [comment]
deprecate works for me, just a note that the samples should follow best practice when/if deprecated, I consider the TodoList the flagship and I believe it uses it as I got in a dependency snarl there if I remember correctly when trying to follow the React deprecated lifecycle re-naming in order to have TodoList work out of the box with no errors/warnings

**Comment by fbartho** [comment]
Yeah, updating TodoList to avoid `reactxp-navigation` is going to be a solid task. :(

If we do deprecate `reactxp-navigation` - I think we should avoid *any* navigation libraries initially, and just tell people: "follow your web-and-mobile-compatible navigation library's instructions, and just use ReactXP components.
